### DISCUSSION TITLE
shame. shame and disgrace.

### DISCUSSION THREAD

**Discussion by gander** [borgmatic-collective/docker-borgmatic#329] [General]
is it beyond the capabilities of the average user, to create a WORKING solution, a tutorial that describes from start to finish how to run a given tool? Why is there so much crap? This is not only true for borgmatic, but for any software that has a community. Those who know how to deal with common problems keep this knowledge to themselves. Terribly frustrating.

**Comment by grantbevis** [comment]
Do you have any specific problem you’d like help with? An error message or some other issue you’d like to raise?

**Comment by witten** [comment]
Pull requests welcome!
